src/pybind/mgr/dashboard/tools.py

```diff
@@ -5,6 +5,7 @@
 import inspect
 import json
 import functools
+import ipaddress
```

Comment:
Don't forget to remove the ``import socket`` line, otherwise you'll get the lint warning ``tools.py:16:0: W0611: Unused import socket (unused-import)``.